content/02.main-text.md

```diff
@@ -29,20 +29,20 @@ Existing platforms work well for editing text and are widely used for scholarly
 However, they lack features that are important for open writing, such as managing a collaborative manuscript and attributing precise credit among many authors (Table @tbl:platforms).
 None of these platforms offer the ability to address thematically related changes together and enable multiple authors to iteratively refine proposed changes.
 
-| Feature | Manubot | Authorea + BibTeX | Overleaf v2 + BibTeX | Google Docs + Paperpile | Word Online<sup>1</sup> | Markdown on GitHub |
```

Comment:
I removed "+ BibTeX" because I think that in the case of authorea it may be inaccurate and in the case of Overleaf it may be assumed. 

It seems like BibTex refers to a program and less so the `.bib` file format. https://tex.stackexchange.com/a/299286 And I am not sure if Authorea uses the BibTeX program to render their citations, but I think it is likely that they do not.

Reply:
Agree